### PR TITLE
Chrome fix for TA Application

### PR DESCRIPTION
**Pull request by bwgoulet**
- This PR is a fix for the existing TA Application's Chrome support. As of one of the latest Chrome updates, the 'mat-label' element will cause the page to crash upon selection. This fix ensures that students can submit the application on Chrome.

### CODE REVIEW

File: frontend/src/app/ta-application/uta-application/uta-application.component.ts [hunk 1]
```diff
@@ -230,6 +239,10 @@ export class UndergradApplicationComponent implements OnInit, OnDestroy {
     this.destroy$.complete();
   }
 
+  ngAfterViewInit() {
+    this.el.nativeElement.parentElement.removeAttribute('aria-owns');
+  }
+
```
[the remark this answers is not in the document]
Comment (KrisJordan) [reply frontend/src/app/ta-application/uta-application/uta-application.component.ts:245]: It's probably worth a comment clarifying what this is doing / why and/or linking out to the StackOverflow thread (or wherever) that led to this.

[the remark this answers is not in the document]
Comment (ajaygandecha) [reply frontend/src/app/ta-application/uta-application/uta-application.component.ts:245]: @bwgoulet

[the remark this answers is not in the document]
Comment (bwgoulet) [reply frontend/src/app/ta-application/uta-application/uta-application.component.ts:245]: I can update the PR to link to the stack overflow once i get back to the computer 👍

[the remark this answers is not in the document]
Comment (ajaygandecha) [reply frontend/src/app/ta-application/uta-application/uta-application.component.ts:245]: @bwgoulet I wonder if adding this to `index.html` as suggested later on [here](https://github.com/angular/components/issues/28905) might be a better solution to apply to every component - not sure though:

```
<script>
  document.addEventListener('DOMNodeInserted', function () {
    const elements = document.querySelectorAll('[aria-owns]');
    elements.forEach(element => {
      element.removeAttribute('aria-owns');
    });
  });
</script>
```

[the remark this answers is not in the document]
Comment (ajaygandecha) [reply frontend/src/app/ta-application/uta-application/uta-application.component.ts:245]: Also @KrisJordan, with some further digging, noticing that `v16+` in Angular seems to not have this issue.

The issue appears on `v15` and lower on Chrome and MS Edge

[the remark this answers is not in the document]
Comment (bwgoulet) [reply frontend/src/app/ta-application/uta-application/uta-application.component.ts]: Just pushed the new fix with the `index.html` script tag, which also references the stack overflow thread @ajaygandecha

**Review by ajaygandecha — APPROVED** [review]
Code LGTM

**Pull request merged**